public/app/features/logs/components/LogRowContextProvider.tsx

```diff
@@ -210,7 +211,15 @@ export const LogRowContextProvider: React.FunctionComponent<LogRowContextProvide
       after: result ? result.errors[1] : undefined,
     },
     hasMoreContextRows,
-    updateLimit: () => setLimit(limit + 10),
+    updateLimit: () => {
+      setLimit(limit + 10);
+
+      const { datasourceType, uid: logRowUid } = row;
+      reportInteraction('grafana_explore_logs_log_context_load_more_clicked', {
+        datasourceType,
+        logRowUid,
```

Comment:
Having a look at this now, could you also add the new limit as a property here? I think that this is also nice information.